cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -133,11 +133,11 @@ addon-dir-create:
 {% endif %}
 
 {% if pillar.get('enable_node_logging', '').lower() == 'true'
-   and pillar.get('logging_destination', '').lower() == 'elasticsearch'
+   and 'logging_destination' in pillar
    and pillar.get('enable_cluster_logging', '').lower() == 'true' %}
 /etc/kubernetes/addons/fluentd-elasticsearch:
```

Comment:
Will this always put the file under fluentd-elasticsearch? If so, does that matter?